MovieViewer/FiltersViewController.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  FiltersViewController.swift
+//  MovieViewer
+//
+//  Created by Dang Quoc Huy on 7/7/16.
+//  Copyright © 2016 Dang Quoc Huy. All rights reserved.
+//
+
+import UIKit
+
+let startYear = 1916
+
+class FiltersViewController: UITableViewController {
+
+    @IBOutlet weak var adultContentSwitch: UISwitch!
+    @IBOutlet weak var releaseYearPicker: UIPickerView!
+    @IBOutlet weak var primaryReleaseYearPicker: UIPickerView!
+    
+    var settings = SearchMovieSettings()
```

Comment:
I think you should not initialize this variable here. This variable is required to be set from another view controller, or nothing will happen. So I would change this to
```
var settings: SearchMovieSettings!
```

That way, if you forget to set the settings of this view controller, at least the program will crash instead of silently failing.